.babelrc

```diff
@@ -5,7 +5,16 @@
   ],
   "plugins": [
     "@babel/plugin-proposal-class-properties",
-    "@babel/plugin-syntax-dynamic-import"
+    "@babel/plugin-syntax-dynamic-import",
```

Comment:
shouldn't this be added to
   
    "env": {
      "production": {
        "plugins": [...]
      }
    }

?

Reply:
@tomegz Do you mean `@babel/plugin-syntax-dynamic-import` or `transform-react-remove-prop-types`? 🤔

Reply:
You're right. `transform-react-remove-prop-types` should be place in `production` part.
I already modified it. Thanks!